crates/relayer/src/chain/cosmos/eip_base_fee.rs

```diff
@@ -21,7 +21,7 @@ pub async fn query_eip_base_fee(
 
     let chain_name = chain_id.name();
 
-    let is_osmosis = chain_name.starts_with("osmosis") || chain_name.starts_with("osmo-test");
```

Comment:
I think we should instead fix how the chain id is parsed. I would expect `osmos-test-5` to be parsed as 

```
ChainId {
  id: "osmo-test",
  version: 5
}
```

@ljoss17 @allthatjazzleo What do you think?

Reply:
I do think so
But not sure will it be a breaking changes to other chain

Reply:
Nevermind, we are parsing the chain id correctly, ie. `osmo-test-5` is parsed as `Chain { id: "osmo-test-5", version: 5 }` but the `name` method is wrong, and does the splitting on `-` the wrong way, which discards the `-test` parts of the chain name.